lib/git_reflow/workflows/core.rb

```diff
@@ -15,8 +15,9 @@ class Core
       command(:setup, defaults: {local: false, enterprise: false}) do |**params|
         reflow_options             = { project_only: params[:local], enterprise: params[:enterprise] }
         existing_git_include_paths = GitReflow::Config.get('include.path', all: true).split("\n")
+        evaluated_config_path = GitReflow::Config::CONFIG_FILE_PATH.gsub(/\$([a-zA-Z_]+[a-zA-Z0-9_]*)|\$\{(.+)\}/) { ENV[$1 || $2] }
```

Comment:
Would need to change some of the rspec tests:
```
rspec ./spec/lib/git_reflow/config_spec.rb:32 # GitReflow::Config.set(key) should have run command silently "git config -f $HOME/.gitconfig.reflow --replace-all chucknorris.roundhouse \"to the face\""
rspec ./spec/lib/git_reflow/config_spec.rb:42 # GitReflow::Config.unset(key) should have run command silently "git config -f $HOME/.gitconfig.reflow --unset-all chucknorris.roundhouse "
rspec ./spec/lib/git_reflow/config_spec.rb:46 # GitReflow::Config.unset(key) for multi-value keys should have run command silently "git config -f $HOME/.gitconfig.reflow --unset-all chucknorris.roundhouse \"to the face\""
```
But it is doable if you'd rather go that route.